composer.json

```diff
@@ -24,28 +24,28 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-console": "^2.6",
```

Comment:
If laminas-console is still required, how should an end user install laminas-test?

Reply:
`--ingnore-platform-req` ? Same like PHP 8.0 is "supported" now. It's definitely last version of 3.x, 4.x without `laminas-console` is waiting for release.

Reply:
> Same like PHP 8.0 is "supported" now. 

Unbeautiful!

> …4.x without laminas-console is waiting for release.

Are you planning a release right after 3.8.0?

Reply:
Yes!